setup.cfg

```diff
@@ -78,6 +78,7 @@ ignore =
 	W503
 	E203
 	E501
+	F401
```

Comment:
Not sure if it's better to do this or specifically exclude `rubicon_ml/client/__init__.py`. There's an issue with the imports in this file falsely being marked as unused.

Reply:
try removing this and doing what I mentioned above - if that doesn't work then this is cool